src/repository/screens/repository.screen.js

```diff
@@ -99,25 +99,46 @@ class Repository extends Component {
       changeStarStatusRepoByDispatch,
       forkRepoByDispatch,
       navigation,
-      username,
     } = this.props;
-    const showFork = repository.owner.login !== username;
 
     if (index === 0) {
+      // Share
+      this.shareRepository(repository);
+    }
+
+    if (index === 1) {
```

Comment:
I know this was already here in this format, but any chance you can just change it to `if`, `else if` blocks instead of multiple separate `if` conditions :O